MesnardBarba-CiSE.md

```diff
@@ -5,11 +5,11 @@
 Our research group prides itself for having adopted Reproducible Research practices. 
 Barba made a public pledge titled *"Reproducibility PI Manifesto"* ^(1) (PI: Principal Investigator), which at the core is a promise to make all research materials and methods open access and discoverable: releasing code, data and analysis/visualization scripts.
 
-In 2014, we published a study on Physics of Fluids titled *"Lift and wakes of flying snakes."* ^(2) 
-It is a study that uses our in-house code for solving the equations of fluid motion in two dimensions (2D), with a solution approach called “immersed boundary method.” 
+In 2014, we (this "we" might suggest that Mesnard and Barba published the paper?) published a study on Physics of Fluids titled *"Lift and wakes of flying snakes."* ^(2) 
```

Comment:
I don't think it's really a source of confusion, since the reference is there with the correct list of authors. And I can't think of another way to write it that is not more awkward, like "one of us with another co-author" (ugh). A royal "we" meaning "us folks in the same group" is fine, I think!